staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -218,11 +230,8 @@ func (o *DeleteOptions) Validate() error {
 		return fmt.Errorf("cannot set --all and --field-selector at the same time")
 	}
 
-	switch {
-	case o.GracePeriod == 0 && o.ForceDeletion:
+	if o.GracePeriod == 0 {
 		fmt.Fprintf(o.ErrOut, "warning: Immediate deletion does not wait for confirmation that the running resource has been terminated. The resource may continue to run on the cluster indefinitely.\n")
-	case o.ForceDeletion:
-		fmt.Fprintf(o.ErrOut, "warning: --force is ignored because --grace-period is not 0.\n")
```

Comment:
Add here validation:
```go
if o.GracePeriod > 0 && o.ForceDeletion {
    return fmt.Errof("--force and --grace-period greater than 0 cannot be specified together")
}
```

Reply:
Good idea, much cleaner, thanks.

Reply:
Agreed. This is a much cleaner change. Thanks